### PR TITLE
Set colors via config

### PR DESCRIPTION
**Pull request by c-a-v-a**

### CODE REVIEW

**Comment by p00f** [comment]
Can you also change the readme?

**Comment by p00f** [comment]
Also, please add `termcolors` https://github.com/p00f/nvim-ts-rainbow/blob/11b070c97b0c3908b85f7459398f0a79401fc411/lua/rainbow/internal.lua#L6

**Comment by c-a-v-a** [comment]
Also I noticed something weird. When I set colors (in colors.lua or in config), they are used out of order. Is this expected behavior, or some kind of bug?

**Comment by p00f** [comment]
Yeah, this is expected because it counts all nodes for calculating depth. #25 had fixed this but there were other problems I didn't know how to handle at the time so I reverted it. Once I get time I'll do #29 and #25

File: README.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ I've used the gruvbox palette because it is nice and subtle. I wanted to use VIB
...
+		},
+		-- Term colors
+		termcolors = {
+			-- Term colors here
+		}
+	},
+}
+```
+
 To change the colours, edit `lua/rainbow/colors.lua`. If you want to override only a *few* colours (you can only change colours 1 through 7 this way), you can do it in your init.vim: (thanks @delphinus !)
```
Comment (p00f) [inline README.md:39]: please delete the first sentence of this line, it is repeated

Comment (p00f) [reply README.md:39]: (Line 39)

File: lua/rainbow/internal.lua [hunk 2]
```diff
@@ -3,11 +3,24 @@ local nvim_query = require("vim.treesitter.query")
 local parsers = require("nvim-treesitter.parsers")
 local configs = require("nvim-treesitter.configs")
 local nsid = vim.api.nvim_create_namespace("rainbow_ns")
-local colors = require("rainbow.colors")
 local termcolors = require("rainbow.termcolors")
```
Comment (p00f) [inline lua/rainbow/internal.lua:6]: This line should be deleted too

Comment (p00f) [reply lua/rainbow/internal.lua:6]: (` local termcolors = require("rainbow.termcolors")`)

File: README.md (hunk 1 above)
Comment (p00f) [reply README.md:39]: (To change the colours, edit `lua/rainbow/colors.lua`.)

**Review by p00f — CHANGES_REQUESTED** [review]

**Review by p00f — APPROVED** [review]

**Comment by p00f** [comment]
Thank you very much!

**Pull request merged**